### PR TITLE
Update async-operation and cumulus-ecs-task references to node 10 versions

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** 

Updates cumulus-ecs-task/async-operations images to their new node 10 versions

Addresses [CUMULUS-1626: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1626)

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: example/app/config.yml [hunk 1]
```diff
@@ -59,7 +59,7 @@ default:
       username: cumulususer
     services:
       EcsTaskHelloWorld:
-        image: cumuluss/cumulus-ecs-task:1.3.0
+        image: cumuluss/cumulus-ecs-task:1.4.0
```
Comment (markdboyd) [inline example/app/config.yml]: Not that anyone is deploying with Kes, but would this version be backwards compatible with a Kes deployment?

File: packages/deployment/app/config.yml [hunk 2]
```diff
@@ -143,7 +143,7 @@ default:
 
     tasks:
       AsyncOperation:
-        image: cumuluss/async-operation:26
+        image: cumuluss/async-operation:27
```
Comment (markdboyd) [inline packages/deployment/app/config.yml]: Not that anyone is deploying with Kes, but would this version be backwards compatible with a Kes deployment?

File: example/app/config.yml (hunk 1 above)
Comment (Jkovarik) [reply example/app/config.yml]: Should be.   And in fact I'm using it in the backport.      What it's *not* compat with is node8 + kes.  

Of course that won't matter for more than a week or so.

File: packages/deployment/app/config.yml (hunk 2 above)
Comment (Jkovarik) [reply packages/deployment/app/config.yml]: Yes.  But not with a kes node-8 deployment.

**Pull request merged**